packages/sku/context/defaultCompilePackages.js

```diff
@@ -1,16 +1,32 @@
-const path = require('path');
+const { posix: path } = require('path');
 const chalk = require('chalk');
 const glob = require('fast-glob');
+const { execSync } = require('child_process');
 
-const { cwd } = require('../lib/cwd');
+const { cwd: skuCwd } = require('../lib/cwd');
+const toPosixPath = require('../lib/toPosixPath');
 
 /** @type {string[]} */
 let detectedCompilePackages = [];
 
 try {
+  const cwd = skuCwd();
+  const gitRepoRoot = execSync('git rev-parse --show-toplevel', { cwd })
+    .toString()
+    .trim();
+
+  const pnpmVirtualStorePath = path.join(
+    toPosixPath(gitRepoRoot),
+    'node_modules/.pnpm',
+  );
+  const pnpmVirtualStoreGlob = path.join(
+    pnpmVirtualStorePath,
+    '@seek*/node_modules/@seek/*/package.json',
+  );
+
   detectedCompilePackages = glob
-    .sync([`node_modules/@seek/*/package.json`], {
-      cwd: cwd(),
+    .sync(['node_modules/@seek/*/package.json', pnpmVirtualStoreGlob], {
```

Comment:
I'd prefer the globs to be as close to each other as possible. Makes it easier to keep them in sync.